.rubocop_local.yml

```diff
@@ -0,0 +1,18 @@
+#
+# Overrides
+#
+GlobalVars:
+  AllowedVariables:
+  # Loggers
+  - $log
```

Comment:
Since this is a pure gem, I should never have $log in here, so this can be removed (I can do it in a follow up).